app/poi_api/poi_errors.py

```diff
@@ -0,0 +1,11 @@
+from app.base_api_error import BaseApiError
+

```

Comment:
Just a suggestion since you're going to use this file structure repeatedly. You could just have a "poi" folder and then under it have api, errors, rest_handler, service etc. to avoid the repetition.